File: SGA_Desktop/SGA_Desktop/Models/TraspasoArticuloDto.cs
namespace SGA_Desktop.Models
{
    public class TraspasoArticuloDto
    {
        public Guid Id { get; set; }
        public string AlmacenOrigen { get; set; }
        public string UbicacionOrigen { get; set; }
        public string AlmacenDestino { get; set; }
        public string UbicacionDestino { get; set; }
        public int UsuarioId { get; set; }
        public DateTime Fecha { get; set; }
        public string CodigoArticulo { get; set; }
        public decimal Cantidad { get; set; }
        public string Estado { get; set; }
        public string Origen => $"{AlmacenOrigen} / {UbicacionOrigen}";
        public string Destino => $"{AlmacenDestino} / {UbicacionDestino}";
    }
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/TraspasoArticuloDto.kt
package com.example.sga.data.dto.traspasos

data class TraspasoArticuloDto(
    val id: String,
    val almacenOrigen: String,
    val ubicacionOrigen: String,
    val almacenDestino: String?,
    val ubicacionDestino: String?,
    val usuarioId: Int,
    val fecha: String,
    val codigoArticulo: String,
    val cantidad: Double,
    val estado: String
) 